cli/package.json

```diff
@@ -141,6 +142,11 @@
       "require": "./react/dist/cypress-react.cjs.js",
       "types": "./react/dist/index.d.ts"
     },
+    "./svelte": {
```

Comment:
I looked into this - these entries actually do not do anything from what I can tell, or, at least, they are not respected by webpack 4. I am pretty sure we don't need these at all.

What's actually happening is the bundlers look at `node_modules/cli/{react,vue}` and look inside for `package.json`, and resolve from there.

Reply:
They're for types and Vite. If you take them out, rebuild, and try to do a deep import from `cypress/vue`, Vite will say that it cannot find that module.